draft-ietf-tls-subcerts.md

```diff
@@ -153,14 +161,14 @@ validity period longer than 7 days.  This mechanism is described in detail in
 It was noted in [XPROT] that certificates in use by servers that support
 outdated protocols such as SSLv2 can be used to forge signatures for
 certificates that contain the keyEncipherment KeyUsage ({{!RFC5280}} section
-4.2.1.3)  In order to prevent this type of cross-protocol attack, we define a
+4.2.1.3).  In order to prevent this type of cross-protocol attack, we define a
 new DelegationUsage extension to X.509 that permits use of delegated
 credentials.  (See {{certificate-requirements}}.)
 
 ## Rationale
 
 Delegated credentials present a better alternative than other delegation
-mechanisms like proxy certificates {{RFC3820}} for several reasons:
+mechanisms like proxy certificates {{?RFC3820}} for several reasons:
```

Comment:
What's this question mark for? Is this a typo, or is this syntax that I've never seen before?

Reply:
{{?RFCXXXXX}} will insert an informative reference and {{!RFCXXXX} will insert a normative reference.  Learned that from martinthomson.

Reply:
Oh neat. Thanks for the tip!